src/backend/columnar/sql/udfs/citus_finish_pg_upgrade/latest.sql

```diff
@@ -0,0 +1,26 @@
+IF version() ~ '12' or version() ~ '13' THEN
```

Comment:
kind of odd to puts this in the udfs/ directory when it is not defining a UDF

Reply:
see comment on `citus_finish_pg_upgrade`